VocaDbModel/Database/Queries/EventQueries.cs

```diff
@@ -632,7 +640,7 @@ public class EventQueries : QueriesBase<IEventRepository, ReleaseEvent> {
 					};
 					session.Save(series);
 
-					var diff = new ReleaseEventSeriesDiff(ReleaseEventSeriesEditableFields.OriginalName | ReleaseEventSeriesEditableFields.Names);

```

Comment:
Could you change this back?

Reply:
I think this shouldn't be changed back because this creates a snapshot just like other entry types.
If I change this back, the "External links" row in the "Content" table is displayed as one column rather than two columns when comparing two revisions because the WebLinks field is not included in the serialized data.

Please take the following steps to confirm the problem.
1. Create a new event series without any external links.
2. Add some external links to the series.
3. Compare these two revisions.